data-migrator/core/src/test/java/io/camunda/migrator/interceptor/InterceptorConfigurationTest.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.migrator.interceptor;
```

Comment:
- [ ] to be moved to `io.camunda.migration.data.qa.history.entity` after rebase
- [ ] imports should be adjusted too